impeller/docs/faq.md

```diff
@@ -100,3 +100,113 @@
     as development progresses.
 * How do you run `impeller_unittests` with Playgrounds enabled?
   * Specify the `--enable_playground` command-line option.
+* Describe Impeller in a Tweet.
+  * "Impeller is AOT mode for rendering."
+* Why did the Flutter team build Impeller?
+  * The short answer, for consistent performance. The longer answer...
+  * Flutter applications, via their use of Skia, were susceptible to the problem
+    of jank due to shader compilation. Such problems were [reported as early as
+    2015](https://github.com/flutter/flutter/issues/813).
+  * Shaders are tiny programs that run on the GPU when a specific combination of
+    rendering intent is requested by the application. Flutters API is extremely
+    expressive and it was thought to be impossible to determine the entire set
```

Comment:
thought to? by whom? thought as in pondered?or concluded?

maybe: "and after analysis, the Flutter team concluded that given Flutter's architecture it was infeasible to know (precompute? determine?) all shaders an app might need at runtime

Reply:
oh, I think here we mean: "during the original design of Flutter's architecture, it was built on an assumption that it was infeasible for to determine ahead of time the entire set of shaders an application might need. Thus, we built the original Flutter architecture assuming we had to create shaders on the fly."

something like that.

Reply:
I reworded it. But it may need to be tinkered on some more. Another pass please?